lib/model_patches.rb

```diff
@@ -10,8 +10,9 @@
   # OutgoingMessage.class_eval do
   #   # Add intro paragraph to new request template
   #   def default_letter
-  #     # this line let the default_letter url parameter to overridden
-  #     # any value which is set here
+  #     # uncomment this line to allow the default_letter text to be replaced
+  #     # by the value supplied by the API
+  #     # e.g. http://demo.alaveteli.org/new/tgq?default_letter=this+is+a+test
```

Comment:
But that's not what this does. There's no line to uncomment to enact the behaviour you describe – that happens automatically. Uncommenting the _method_ allows the API behaviour to happen, as well as adding an "intro paragraph to new request template" when the API param is not specified.